proposals/0186-template-titles.md

```diff
@@ -30,6 +30,15 @@ Basic templates have nothing to orient the user to where they are in the UI. A t
         </description>
     </param>
 </function>
+
+<enum name="TextFieldName" since="1.0">
+    <!-- Existing RPCs -->
+
+    <!-- Additions -->
+    <element name="templateTitle" since="x.x">
+        <description>The template title field in Show</description>
```

Comment:
Might be better to change `in Show` to `; applies to Show` to make it more consistent with the other elements in `TextFieldName`

Reply:
Fixed